src/main/scala/ru/d4team/chat/config/PostgresConfig.scala

```diff
@@ -1,29 +1,33 @@
 package ru.d4team.chat.config
 
-import zio._
-import zio.sql.HikariConnectionPoolConfig
+import org.flywaydb.core.Flyway
+import org.flywaydb.core.api.configuration.FluentConfiguration
 
 final case class PostgresConfig(
-    host: String,
-    port: Int,
-    db: String,
-    user: String,
-    password: String
+  host: String,
+  port: Int,
+  db: String,
+  user: String,
+  password: String,
+  migrationsLocation: String,
+  schema: String
 ) {
   val url = s"jdbc:postgresql://$host:$port/$db"
 }
 
 object PostgresConfig {
-
-  val hikariConnectionPoolConfig: RLayer[PostgresConfig, HikariConnectionPoolConfig] = ZLayer {
-    for {
-      pgConfig  <- ZIO.service[PostgresConfig]
-      poolConfig = HikariConnectionPoolConfig(
-                     url = pgConfig.url,
-                     userName = pgConfig.user,
-                     password = pgConfig.password
-                   )
-    } yield poolConfig
-  }
```

Comment:
Why was it removed? I think it will be needed to create DAO services.

Reply:
yep, i will undo this deletion

Reply:
corrected